src/tests/ftest/cart/test_group_np_cli.c

```diff
@@ -30,11 +30,55 @@
 #include <assert.h>
 #include <getopt.h>
 #include <semaphore.h>
+#include <ctype.h>
 
 #include "tests_common.h"
 #include "test_group_rpc.h"
 #include "test_group_np_common.h"
 
+
+static void
+send_rpc_shutdown(crt_endpoint_t server_ep, crt_rpc_t *rpc_req)
+{
+	int rc = crt_req_create(test_g.t_crt_ctx[0], &server_ep,
+					CRT_PROTO_OPC(TEST_GROUP_BASE,
+						      TEST_GROUP_VER, 1), &rpc_req);
+	D_ASSERTF(rc == 0 && rpc_req != NULL,
+		  "crt_req_create() failed. "
+			"rc: %d, rpc_req: %p\n", rc, rpc_req);
+	rc = crt_req_send(rpc_req, client_cb_common, NULL);
+	D_ASSERTF(rc == 0, "crt_req_send() failed. rc: %d\n", rc);
+
+	tc_sem_timedwait(&test_g.t_token_to_proceed, 61, __LINE__);
+}
+
+static void
+send_rpc_swim_check(crt_endpoint_t server_ep, crt_rpc_t *rpc_req)
+{
+	struct test_swim_status_in	*rpc_req_input;
+	/*struct test_swim_status_out	*rpc_req_output; */
+
+	int rc = crt_req_create(test_g.t_crt_ctx[0], &server_ep,
+					CRT_PROTO_OPC(TEST_GROUP_BASE,
+						      TEST_GROUP_VER, 2), &rpc_req);
```

Comment:
(style)  line over 80 characters

Reply:
[Vim `ruler`](http://vimdoc.sourceforge.net/htmldoc/options.html#'ruler') lists this line as 48 chars long, and a vim regex doesn't find any code lines (outside of the copyright notice at the top) exceeding 70 chars (`/\v^.{70,}`) in `test_group_np_cli.c`.